### PR TITLE
Implemented the mobile and desktop version of a web course 

### PR DESCRIPTION
**Pull request by vangelif**
### In this PR:

- Deployed a home and an about page in both desktop and mobile
- Speakers are dynamically inserted with the use of JS
- Followed HTML/CSS/JS best practises
- Paid tribute to the designer and artists for their work
- Run with no error all linters
- Recorded the explanatory video that you can find by [clicking here](https://bit.ly/40MpSD5)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,178 @@
...
+- **[Dynamically inserted Speakers]** 
+- **[Implemented mobile and desktop versions]**
+- **[Demonstration of home and about pages]** 
+- **[Interactive Mobile menu]**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="https://vangelif.github.io/sinology-course/">Check out the live demo</a> 
+
+- <a href="https://bit.ly/3RRO5Ec">Check out the live video</a>
```
Comment (ghost) [inline README.md]: - [x] Good video presentation but I would like to encourage you to stick to the project requirement as the video was supposed to cover a maximum length of `5 minutes`.

File: README.md [hunk 2]
```diff
@@ -1 +1,178 @@
...
+### Key Features <a name="key-features"></a>
+
+- **[Dynamically inserted Speakers]** 
+- **[Implemented mobile and desktop versions]**
+- **[Demonstration of home and about pages]** 
+- **[Interactive Mobile menu]**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="https://vangelif.github.io/sinology-course/">Check out the live demo</a>
```
Comment (ghost) [inline README.md:63]: - [x] I noticed that the `deployed`  version of your project is a little bit different from the version I cloned locally.  Apparently, the version I have locally contains a mobile menu on the desktop version. Kindly checkout on that.

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+      <div class="intro-bottom">
+        <h3><strong>2023.02.06(MON) ~ 09(THU)</strong></h3>
+        <aside>@ National Museum of China, Art Center Nabi and more</aside>
+      </div>
+    </section>
+    <!-- main program -->
+    <section class="program">
+      <h3>Main Program</h3>
+      <hr />
+      <ul>
```
Comment (ghost) [inline index.html:105]: - [x] Kindly position your `main programs` title correctly. Currently, it is partly visible.

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+        <li>
+          <span><i class="fas fa-robot"></i></span>
+          <h3>Lecture</h3>
+          <p>
+            Listen to the speakers Lorem, ipsum dolor. Laboriosam nemo rerum
+            quod praesentium eni.
+          </p>
+        </li>
+        <li>
```
Comment (ghost) [inline index.html]: - [x] Kindly ensure that the title of your `program cards` are on the same level as in the original design and add `hover effect` to them.

image

### Checkout the original design

image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+          <p>SC Global Summit 2013 in Shanghai</p>
+        </div>
+
+        <div class="past-item-2">
+          <img src="images/past-events/beijing.jpg" />
+          <h2>2011</h2>
+          <p>SC Global Summit 2013 in Beijing</p>
+        </div>
+      </div>
+    </section>
```
Comment (ghost) [inline about.html:136]: - [x] Kindly position the `summit cards` correctly on your `about` page as they appear to overflow towards the footer section blocking its content.

image

- [x] This also affected the mobile version of your page

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @vangelif , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @vangelif 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x]  Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**